src/lib/api/entity-api.ts

```diff
@@ -193,6 +193,7 @@ export namespace EntityApi {
     url: string,
     config?: any
   ): Promise<R> => {
+    console.log("deleteFn", url, config);
```

Comment:
```suggestion
```

Reply:
Removed in accebcf99c476dcd909eb3b27c415223ebecd387